Tasks/IISWebAppManagementOnMachineGroup/task.json

```diff
@@ -627,7 +627,8 @@
 		"InvalidVirtualPath": "Virtual path should begin with a /",
 		"InvalidIISDeploymentType": "Invalid IIS Deployment Type : {0}",
 		"InvalidActionIISWebsite": "Invalid action '{0}' selected for the IIS Website.",
-		"InvalidActionIISAppPool": "Invalid action '{0}' selected for the IIS Application Pool."
+		"InvalidActionIISAppPool": "Invalid action '{0}' selected for the IIS Application Pool.",
+		"SSLCertWarningInvalidCharacters": "SSL Certificate thumbprint contains non-hexadecimal characters. Trimming all non-hexadecimal characters."
```

Comment:
it will be good to add a forward link to documentation here and help-markdown https://msdn.microsoft.com/en-us/library/windows/desktop/cc307220(v=vs.85).aspx